packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -57,60 +57,65 @@ describe('background sync queue test', () => {
     await backgroundSyncQueue.cleanupQueue();
   });
 
-  it('check defaults', () => {
+  it('should get default values if nothing is given in constructor', () => {
     const defaultsBackgroundSyncQueue
       = new workbox.backgroundSync.test.BackgroundSyncQueue({});
-    chai.assert.isObject(defaultsBackgroundSyncQueue._queue);
-    chai.assert.isObject(defaultsBackgroundSyncQueue._requestManager);
-    chai.assert.equal(defaultsBackgroundSyncQueue._queue._queueName,
-      workbox.backgroundSync.test.Constants.defaultQueueName + '_1');
-    chai.assert.equal(defaultsBackgroundSyncQueue._queue._config.maxAge,
-      workbox.backgroundSync.test.Constants.maxAge);
-    chai.assert.equal(
-      JSON.stringify(
-        defaultsBackgroundSyncQueue._requestManager._globalCallbacks),
-      JSON.stringify({}));
+    expect(defaultsBackgroundSyncQueue._queue).to.be.an('object');
+    expect(defaultsBackgroundSyncQueue._requestManager).to.be.an('object');
+    expect(defaultsBackgroundSyncQueue._queue._queueName).to.be
```

Comment:
Use `to.equal()` instead of `to.be.equal()` (here and throughout) as the latter doesn't read well.